src/bidiServer/BrowserInstance.ts

```diff
@@ -112,6 +112,7 @@ export class BrowserInstance {
     const browserProcess = launch({
       executablePath,
       args: chromeArguments,
+      env: process.env,
```

Comment:
Needed to pass this locally to make xvfb work all the time, not sure how it worked before.